src/components/dashboard/main-content/DashboardMainContent.jsx

```diff
@@ -6,8 +6,19 @@ import { Button } from '@edx/paragon';
 import { CourseEnrollments } from './course-enrollments';
 
 const DashboardMainContent = () => {
-  const { enterpriseConfig: { name, slug } } = useContext(AppContext);
+  const { enterpriseConfig: { name, slug, disableSearch } } = useContext(AppContext);
 
+  if (disableSearch) {
```

Comment:
Could this be misleading to say "you are not enrolled in any courses" when `disableSearch` is true when they might actually have enterprise enrollments? `disableSearch` and not having enrollments are not mutually exclusive.

Should the learner end up on the Dashboard when `disableSearch` is true, I think it still makes sense to show their existing enrollments?

Would it make sense to have this messaging conditionally rendered as `children` of `CourseEnrollments` below where the existing "Find a Course" empty state lives?

e.g.

```jsx
<CourseEnrollments>
  {disableSearch ? (
    <>
      <h2>Get Started Learning</h2>
      <p>
        You are not enrolled in any courses sponsored by {name}.
        Reach out to your administrator to begin learning with edX!
      </p>
    </>
  ) : (
    <>
      <h2>Find a Course</h2>
      <p>
        You are not enrolled in any courses sponsored by {name}.
        To start taking a course, browse the catalog below.
      </p>
      {/* ... */}
    </>
  )}
</CourseEnrollments>
```

Reply:
Ah, yes, good point.